packages/zambdas/src/ehr/incomplete-encounters-report/index.ts

```diff
@@ -0,0 +1,295 @@
+        return false;
+      }
+
+      // Get visit status
+      const visitStatus = getVisitStatus(appointment, encounter, true);
+
+      // Terminal states that should be excluded from the report
+      const terminalStates = ['completed', 'cancelled', 'no-show'];
+
+      return !terminalStates.includes(visitStatus);
+    });
+
+    console.log(`Found ${incompleteEncounters.length} incomplete encounters`);
+
+    // Build the response data
+    const encounterItems = incompleteEncounters.map((encounter) => {
+      const appointmentRef = encounter.appointment?.[0]?.reference;
+      const appointment = appointmentRef ? appointmentMap.get(appointmentRef) : undefined;
+      const patientRef = encounter.subject?.reference;
+      const patient = patientRef ? patientMap.get(patientRef) : undefined;
+
+      // Get location name from Location resource
+      const locationRef = appointment?.participant?.find((p) => p.actor?.reference?.startsWith('Location/'))?.actor
+        ?.reference;
+      const location = locationRef ? locationMap.get(locationRef) : undefined;
+      const locationName = location?.name || 'Unknown';
+      const locationId = locationRef ? locationRef.replace('Location/', '') : undefined;
+
+      // Get attending practitioner name
+      const attendingPractitionerId = getAttendingPractitionerId(encounter);
+      const attendingPractitioner = attendingPractitionerId ? practitionerMap.get(attendingPractitionerId) : undefined;
+      const attendingProviderName = attendingPractitioner
+        ? `${attendingPractitioner.name?.[0]?.given?.[0] || ''} ${attendingPractitioner.name?.[0]?.family || ''}`.trim()
+        : 'Unknown';
+
+      // Determine visit type based on appointment meta tags
+      const visitType = appointment?.meta?.tag?.some((tag) => tag.code === OTTEHR_MODULE.TM)
+        ? 'Telemed'
+        : appointment?.meta?.tag?.some((tag) => tag.code === OTTEHR_MODULE.IP)
+        ? 'In-Person'
+        : 'Unknown';
+
+      const visitStatus = appointment ? getVisitStatus(appointment, encounter, true) : 'unknown';
+
+      return {
+        appointmentId: appointment?.id || '',
+        patientId: patient?.id || '',
+        patientName: patient ? `${getPatientFirstName(patient)} ${getPatientLastName(patient)}`.trim() : 'Unknown',
+        dateOfBirth: patient?.birthDate || '',
+        visitStatus,
+        appointmentStart: appointment?.start || '',
+        appointmentEnd: appointment?.end || '',
+        location: locationName || 'Unknown',
+        locationId,
+        attendingProvider: attendingProviderName,
+        visitType,
+        reason: encounter.reasonCode?.[0]?.text || appointment?.appointmentType?.text || '',
+      };
+    });
+
+    const response: IncompleteEncountersReportZambdaOutput = {
+      message: `Found ${encounterItems.length} incomplete encounters`,
+      encounters: encounterItems,
+    };
+
+    // console.log('Response:', response);
+
```

Comment:
i think logging this could cause our zambda logs to bloat quickly, so i agree to leave it out.

```suggestion
```